src/static/js/modules/secondary-nav-toggle.js

```diff
@@ -25,11 +26,10 @@ function _navSecondaryToggle() {
 
 // Tests whether or not the secondary nav should be toggled.
 function _navSecondaryToggleTest() {
-  var isSmall =
-    $( '.nav-secondary .nav-secondary_link__button' ).is( ':visible' );
+  var isSmall = getBreakpointState().isSmall;
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/7151afaa582863e6c8776a3f773c280a9ba94047